packages/contentstack-migration/examples/05-Update-reference-entry-from-mapper.js

```diff
@@ -0,0 +1,138 @@
+const fs = require('fs');
+const chalk = require('chalk');
+module.exports = async ({ migration, stackSDKInstance, managementAPIClient, config }) => {
+  const modules = ['entries', 'assets', 'extensions'];
+
+  const readAllModulesUids = (filePath) => {
+    let uidMapping = [];
+    modules.forEach((module) => {
+      if (fs.existsSync(`${filePath}/mapper/${module}/uid-mapping.json`)) {
```

Comment:
@cs-raj for path Concatenation pls use path.join or resolve instead of doing it manually

Reply:
Got it Added